modules/ROOT/pages/connected-apps-permissions.adoc

```diff
@@ -0,0 +1,80 @@
+= Connected Apps Permissions by User
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+
+Connected Apps can be used by organization administrators, developers, and end users; each of these roles has different privileges when connecting applications,
+and there are some restrictions to ensure sensitive data remains secure.
+
+== Organization Administrator
+
+Organization administrators are the only users that can view and manage connected apps in Access Management.
+
+To manage your connected apps, navigate to Access Management and select the *Applications* tab.
+
+// screenshot of Connected Apps page
+
+The *Connected Apps* page shows a list of apps that are connected to Anypoint Platform.
+To add an application, select *Add Application*.
+
+=== Create an App
+
+When you create an app as an organization administrator, consider the below:
+
+* An organization can own up to 200 connected apps.
+* You can disable or whitelist connected apps.
+* When you create an app, a clientID is assigned, which can't be modified. However, you can modify client secrets in the *Application Settings* page.
+* When you create an application with the OAuth client credential grant type, the app is owned by the organization; any organization administrator can control the app.
+* For apps that use the user grant type, actions taken by the app are logged with an application ID and user ID in the audit log so administrators can monitor and audit data use.
```

Comment:
there's no "user grant type". There are several grant types that an application can use to act on behalf of a user: JWT Bearer, authorization_code, and basically every grant type that's not client_credentials